IaC/cloud.cd/init.groovy.d/cloud.groovy

```diff
@@ -94,16 +94,18 @@ initMap['docker'] = '''
         echo try again
     done
 
+    sudo amazon-linux-extras install epel -y
+    sudo yum -y install java-1.8.0-openjdk git docker p7zip
+    sudo yum -y remove java-1.7.0-openjdk
+
     until curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "/tmp/awscliv2.zip"; do
         sleep 1
         echo try again
     done
 
-    unzip /tmp/awscliv2.zip -d /tmp
+    7za -o/tmp x /tmp/awscliv2.zip 
```

Comment:
why do we need p7zip? According to official doc unzip should work
https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2-linux.html

Reply:
Yep, but as we discussed on DS, few runners stuck on `Archive: ` stage, although instance is not removed from AWS
And it started to stuck after `unzip` command